src/api/app/views/webui2/webui/project/show.html.haml

```diff
@@ -48,3 +55,19 @@
         (#{ @comments.length})
       .card-body#comments
         = render partial: 'webui2/webui/comment/show', locals: { commentable: @project }
+
+-# TODO: This is ugly...
```

Comment:
Open to suggestions to improve this...

Reply:
@dmarcoux It doesn't make sense to have this comment, if we can't find a nicer way for this right now. Let's drop it.

Reply:
I agree, I just hoped someone would have a better way to do this. I removed the comment now.